### ISSUE TITLE
Have you considered submitting this to Pypi?

### ISSUE THREAD

**Issue by xjdeng** [bckenstler/CLR#7]
It's cool that you've implemented a cyclical learning rate for Keras, but have you considered adding this to Pypi?  That way, it's a lot easier for others to incorporate CLR in their own repos

**Comment by maxfrei750** [comment]
Another option might be to propose it to the Keras team, so that it can be incorporated as another callback. Just think about it... making a notable contribution to Keras... :open_mouth:

EDIT: I just saw, that this was already proposed in #1 .

**Comment by xjdeng** [comment]
Even better.. Or Tensorflow (since 2.0 now uses the Keras API). I might actually switch from fastai if this change were made.

**Comment by YumainOB** [comment]
[This comment](https://github.com/bckenstler/CLR/issues/1#issuecomment-526159324) makes a link to the [keras-contrib callback](https://github.com/keras-team/keras-contrib/blob/master/keras_contrib/callbacks/cyclical_learning_rate.py)  where bckenstler contributes
